client/my-sites/domains/domain-management/dns/index.jsx

```diff
@@ -33,6 +37,26 @@ const Dns = React.createClass( {
 		return { addNew: true };
 	},
 
+	isMappedDomain() {
+		const domainInfo = find( this.props.domains.list, ( domain ) => {
```

Comment:
We [have `getSelectedDomain`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/domains/index.js#L119) for the same purpose :)